core/container/src/test/java/org/wildfly/swarm/internal/FileSystemUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package org.wildfly.swarm.internal;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Optional;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class FileSystemUtilsTest {
+
+    @Test
+    public void testMavenArgsPresent() {
+        String cmd = "clean -f pom.xml install";
```

Comment:
It only tests the `MavenArgsParser` that gives access to tokenised values. In this case anything passed to `-f`. There is no "fallback" in the parser itself.